src/main/java/com/google/alpollo/SentimentServlet.java

```diff
@@ -17,11 +17,16 @@
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
ideally the `doPost` method shouldn't throw an exception. It should handle all the Exceptions in a try/catch block and return a proper error like you're doing in lines 27-30. You can probably remove the `throws` declaration here. If there is another source that can throw this exception, it should be handled as well

Reply:
I can't simply remove it, because sendError method can throw IOException and I call this method in the catch block, looks like recursion 😆 

sendError method has `throws IOException`, so I think that's the reason why we can get rid of `ServletException`, but not of `IOException` while overriding `doPost`

Or there's still a way to do it?